docs/conf.py

```diff
@@ -54,4 +53,4 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+# html_static_path = ["_static"]
```

Comment:
For reviewers:  I commented out the highlighted line because it _seems_ we're not making use of the `_static` path.  With the change, the following warning _no longer_ occurs on `make html`:

```
WARNING: html_static_path entry '_static' does not exist
````

If there's a reason to keep the configuration, let me know and I'll revert.